README.md

```diff
@@ -153,11 +153,11 @@ Note that in the following examples, I used `efmt-v0.1.0` and `erlfmt-v1.0.0`.
 ### Formatting style
 
 I think the formatting style of `efmt` is much different from `erlfmt`.
-IMO, this is a major point to decide which one should you choose.
+IMO, this is a major point when you decide which one you should choose.
```

Comment:
In a subordinate clause, [subject–auxiliary inversion](https://en.wikipedia.org/wiki/Subject%E2%80%93auxiliary_inversion) does not happen.